diff --git a/parser/y.go b/parser/y.go
index 2462492..6f73610 100644
--- a/parser/y.go
+++ b/parser/y.go
@@ -2148,15 +2148,15 @@ yydefault:
 		{
 		}
 	case 314:
-		yyVAL.type = yyS[yypt-0].type
+		yyVAL.typ = yyS[yypt-0].typ
 	case 315:
-		yyVAL.type = yyS[yypt-0].type
+		yyVAL.typ = yyS[yypt-0].typ
 	case 316:
-		yyVAL.type = yyS[yypt-0].type
+		yyVAL.typ = yyS[yypt-0].typ
 	case 317:
-		yyVAL.type = yyS[yypt-0].type
+		yyVAL.typ = yyS[yypt-0].typ
 	case 318:
-		yyVAL.type = yyS[yypt-0].type
+		yyVAL.typ = yyS[yypt-0].typ
 	case 319:
 		//line go.y:1191
 		{
